sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2640,38 +2639,78 @@ def check_resources_fit_cluster(self, handle: CloudVmRayResourceHandle,
                 launched_resources)
             mismatch_str = ('To fix: use accelerators/number of nodes that can '
                             'be satisfied by the local cluster')
-        # Requested_resources <= actual_resources.
-        # Special handling for local cloud case, which assumes a cluster can
-        # be heterogeneous. Here, launched_resources is a list of custom
-        # accelerators per node, and Resources.less_demanding_than determines
-        # how many nodes satisfy task resource requirements.
-        if not (task.num_nodes <= handle.launched_nodes and
-                task_resources.less_demanding_than(
-                    launched_resources, requested_num_nodes=task.num_nodes)):
-            if (task_resources.region is not None and
-                    task_resources.region != launched_resources.region):
+
+        if len(task.resources) > 1:
+
+            valid_resource = None
+            requested_resource_list = []
+            for resource in task.resources:
```

Comment:
I'm not super sure whether the validation is safe. cc @Michaelvll

Reply:
Updated